src/test/test_group.c

```diff
@@ -323,6 +323,14 @@ static struct crt_proto_format my_proto_fmt_test_group2 = {
 	.cpf_base = TEST_GROUP_BASE,
 };
 
+int my_func(crt_context_t ctx, crt_rpc_t *rpc, void (rpc_handler)(void *), void *arg) {
```

Comment:
(style)  line over 80 characters
(style)  open brace '{' following function declarations go on the next line